src/arrayMethodSort.js

```diff
@@ -4,9 +4,35 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareStrings) {
+    let count = 0;
+
+    do {
+      count = 0;
+
+      for (let i = 0; i < this.length - 1; i++) {
+        if (compareFunction(this[i], this[i + 1]) > 0) {
+          [this[i], this[i + 1]] = [this[i + 1], this[i]];
+          count++;
+        }
+      }
+    } while (count > 0);
+
+    return this;
   };
 }
 
+const compareStrings = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  } else if (stringA === stringB) {
+    return 0;
+  }
```

Comment:
```suggestion
  }
  
  if (stringA === stringB) {
    return 0;
  }
```

Reply:
If an if block always executes a return statement, the subsequent else block is unnecessary. A return in an else if block following an if block that contains a return can be separated into multiple if blocks

More details here
mate-academy.github.io/style-guides/javascript.html#blocks--no-else-return

Reply:
Thanks, i missed it )
Done